src/components/InstancesList/InstancesList.js

```diff
@@ -298,7 +299,10 @@ class InstancesList extends React.Component {
   }
 
   renderNavigation = () => (
-    <FilterNavigation segment={this.props.segment} onChange={this.refocusOnInputSearch} />
+    <>
+      <SearchModeNavigation />
```

Comment:
I suggest to not show this navigation on instances list for now as there is a chance we won't complete whole feature in Nolana

Reply:
Yes, it's reasonable. I commented out this line until Browse is refactored